packages/data-point-codemods/LICENSE

```diff
@@ -0,0 +1,203 @@
+Copyright 2017 Viacom Inc.
```

Comment:
![](http://i0.kym-cdn.com/entries/icons/original/000/007/784/what-year-is-it-robin-williams.jpg)